.ci/Jenkinsfile

```diff
@@ -27,6 +27,10 @@ pipeline {
   }
   parameters {
     string(name: 'GO_VERSION', defaultValue: "1.12.7", description: "Go version to use.")
+    string(name: 'PR', defaultValue: "", description: "Addicional build options to passing compose.py")
```

Comment:
`compose.py` is a bad copy&paste, right?

Reply:
```suggestion
    string(name: 'PR', defaultValue: '', description: "PR to be tested out")
```